docs/volto-light-theme/creating-new-project.md

```diff
@@ -7,64 +7,91 @@ myst:
 ---
 
-# Creating New Project
+# Creating a New Project with Plone and Volto
 
-First you have to  install the cookieplone
-```
-pipx install cookieplone
+Follow the steps below to create a new project using Plone 6 and Volto. This guide assumes you have `pipx` installed.
 
-```
+### Step 1: Install CookiePlone
 
-After that you have to run it to create the project.
+First, install `cookieplone`, a tool for setting up new Plone projects, using `pipx`:
 
+```bash
+pipx install cookieplone
 ```
+
+### Step 2: Create the Project
+
+Once `cookieplone` is installed, run the command below to start the project creation process:
+
+```bash
 pipx run cookieplone
 ```
 
-It is going to ask you multiple question. If you happy with default keep pressing enter. Otherwise overwrite this according to your needs.
-
+You’ll be prompted with several questions. You can accept the default values by pressing **Enter** or overwrite them according to your preferences. Here are the prompts you’ll encounter:
 
 ```
- [1/17] Project Title (Project Title): my-vlt-project
-  [2/17] Project Description (A new project using Plone 6.): My Project
-  [3/17] Project Slug (Used for repository id) (my-vlt-project): my-project
-  [4/17] Project URL (without protocol) (my-project.example.com): my-project.example.com
-  [5/17] Author (Plone Foundation): <your name>
-  [6/17] Author E-mail (collective@plone.org):
-  [7/17] Should we use prerelease versions? (No):
-  [8/17] Plone Version (6.0.13):
-  [9/17] Volto Version (18.0.0):
-  [10/17] Python Package Name (my.project):
-  [11/17] Volto Addon Name (volto-my-project):
+[1/17] Project Title (Project Title): my-vlt-project
+[2/17] Project Description (A new project using Plone 6.): My Project
+[3/17] Project Slug (Used for repository id) (my-vlt-project): my-project
+[4/17] Project URL (without protocol) (my-project.example.com): my-project.example.com
+[5/17] Author (Plone Foundation): <your name>
+[6/17] Author E-mail (collective@plone.org):
+[7/17] Should we use prerelease versions? (No):
+[8/17] Plone Version (6.0.13):
+[9/17] Volto Version (18.0.0):
+[10/17] Python Package Name (my.project):
+[11/17] Volto Addon Name (volto-my-project):
 ```
 
-After creating project you have to build the volto project.
+- **Project Title**: The title of your project (e.g., `my-vlt-project`).
+- **Project Description**: A short description of your project.
+- **Project Slug**: A repository-friendly identifier for your project (e.g., `my-project`).
+- **Project URL**: The URL where your project will be hosted (e.g., `my-project.example.com`).
+- **Author**: Your name.
+- **Author E-mail**: Your email address (can be left blank).
+- **Plone Version**: The Plone version you’re using (e.g., `6.0.13`).
+- **Volto Version**: The Volto version to be used (e.g., `18.0.0`).
+- **Python Package Name**: The Python package name (e.g., `my.project`).
+- **Volto Addon Name**: The Volto add-on name (e.g., `volto-my-project`).
```

Comment:
The above is duplicitous of the console output. I would remove it.

Reply:
done